persistence/src/main/java/org/cbioportal/genome_nexus/persistence/internal/SourceInfoRepository.java

```diff
@@ -30,8 +37,24 @@ public GenomeNexusInfo genomeNexusInfo(@Value("${genomenexus.server.version:NA}"
     }
 
     @Bean
-    public VEPInfo vepInfo(@Value("${gn_vep.server.version:NA}") String serverVersion, @Value("${gn_vep.cache.version:NA}") String cacheVersion, @Value("${gn_vep.region.url:}") String vepRegionURL) {
-        return new VEPInfo(serverVersion, cacheVersion, vepRegionURL);
+    public VEPInfo vepInfo(@Value("${vep.url:}") String vepUrl, @Value("${vep.static:true}") boolean isStatic) throws MalformedURLException {
+        URL url = new URL(vepUrl);
+        RestTemplate restTemplate = new RestTemplate();
+        String serverVersion = NA_STRING;
+        String vepVersion = NA_STRING;
+        String comment = null;
+        try {
+            VepVersionObject vepVersionObject = restTemplate.getForObject(url.getProtocol() + "://" + url.getAuthority() + "/info/software", VepVersionObject.class);
```

Comment:
Can't find where /info/software is.
Also I'm not sure if this is a good practice to call API in repository layer, maybe good to do it in service?